openreview/conference/templates/submissionProcess.js

```diff
@@ -3,10 +3,16 @@ function() {
 
   var SHORT_PHRASE = '';
 
+  var authorMessage = 'Your submission to ' + SHORT_PHRASE + ' has been posted.\n\nSubmission Number: ' + note.number + '\n\nTitle: ' + note.content.title;
+  if (note.content.abstract) {
+    authorMessage += '\n\nAbstract: ' + note.content.abstract;
+  }
+  authorMessage += '\n\nTo view your submission, click here: ' + baseUrl + '/forum?id=' + note.forum + '\n\nIf you are not an author for this submission and want yourself removed, you can do so by reaching out to the person who added you at ' + note.tauthor;
```

Comment:
I would suggest that we change the wording to "If you are not an author of this submission and would like  to be removed, please contact the author who added you at..."

Reply:
thanks for the suggestion. Change has been made.